setup.py

```diff
@@ -174,15 +175,26 @@ def finalize_options(self):
 
     def run(self):
         import os
-        os.system("find . -name '*.pyc' | xargs rm -f")
-        os.system("rm -f python-build-stamp-2.4")
-        os.system("rm -f MANIFEST")
-        os.system("rm -rf build")
-        os.system("rm -rf dist")
-        os.system("rm -rf doc/_build")
-        os.system("rm -f sample.tex")
-
-
+        
+        for root, dirs, files in os.walk(os.path.dirname(os.path.realpath(__file__))):
+            for file in files:
+                if file.endswith('.pyc') and os.path.isfile:
+                    os.remove(os.path.join(root, file))
+
+        curr_dir = os.getcwd()
+        os.chdir(os.path.dirname(os.path.realpath(__file__)))
```

Comment:
Just being picky at this point, but there isn't any reason to repeat `os.path.dirname(os.path.realpath(__file__)` twice.